docs/streams/developer-guide.html

```diff
@@ -185,6 +185,7 @@
         <li>In the <code>init</code> method, schedule the punctuation every 1 second and retrieve the local state store by its name "Counts".</li>
         <li>In the <code>process</code> method, upon each received record, split the value string into words, and update their counts into the state store (we will talk about this feature later in the section).</li>
         <li>In the scheduled <code>punctuate</code> method, iterate the local state store and send the aggregated counts to the downstream processor, and commit the current stream state.</li>
+        <li>When done with the <code>KeyValueIterator&lt;String, Long&gt;</code> we <em>MUST</em> close the iterator, as show above or using the try-with-resources statement (available since JDK7).</li>
```

Comment:
so we need `<em>` and all upper case for `MUST` ? Seems a little bit too much for me. I would not use all upper-case in general.

Reply:
So we need to point out JDK7 ? Streams does not support any older version anyway?

Reply:
Also, `we` -> `you` ?